### ISSUE TITLE
Replace explicit '0' stream with device::get_default_stream() in cudaEventRecord() 

### ISSUE THREAD

**Issue by bjoo** [lattice/quda#1087]
Something I came accross during merging. In places like blas_test.cpp (lines 368, 532) also in dslash_test.cpp and dslash_ctest.cpp  `cudaEventSynchronize` makes reference still to stream=0, in 
places like
```
cudaEventRecord(start,0);
```
under the new scheme these should probably be:
```
cudaEventRecord(start, device::get_default_stream());
```

**Comment by bjoo** [comment]
Also in `multigrid_benchmark_test.cpp:`

**Comment by bjoo** [comment]
also in `covdev_test.cpp`, `staggered_dslash_ctest.cpp` and `staggered_dslash_test.cpp`

**Comment by bjoo** [comment]
Fixed in PR #1088

**Comment by bjoo** [comment]
PR #1088 has been merged so I will close this..
